arlo-client/src/components/MultiJurisdictionAudit/Setup/__snapshots__/index.test.tsx.snap

```diff
@@ -1669,1257 +1669,11 @@ exports[`Setup renders Opportunistic Contests stage with processing next stage 1
-            <option
-              label="West Virginia"
-              value="WV"
-            >
-              West Virginia
-            </option>
-            <option
-              label="Wisconsin"
-              value="WI"
-            >
-              Wisconsin
-            </option>
-            <option
-              label="Wyoming"
-              value="WY"
-            >
-              Wyoming
-            </option>
-          </select>
-          <span
-            class="bp3-icon bp3-icon-double-caret-vertical"
-            icon="double-caret-vertical"
-          >
-            <svg
-              data-icon="double-caret-vertical"
-              height="16"
-              viewBox="0 0 16 16"
-              width="16"
-            >
-              <desc>
-                double-caret-vertical
-              </desc>
-              <path
-                d="M5 7h6a1.003 1.003 0 00.71-1.71l-3-3C8.53 2.11 8.28 2 8 2s-.53.11-.71.29l-3 3A1.003 1.003 0 005 7zm6 2H5a1.003 1.003 0 00-.71 1.71l3 3c.18.18.43.29.71.29s.53-.11.71-.29l3-3A1.003 1.003 0 0011 9z"
-                fill-rule="evenodd"
-              />
-            </svg>
-          </span>
-        </div>
-      </label>
-      <div
-        class="sc-bZQynM ibvdum"
-      >
-        <div
-          class="sc-gzVnrw kvvweo"
-        >
-          Click "Browse" to choose the appropriate file from your computer. This file should be a comma-separated list of all the jurisdictions participating in the audit, plus email addresses for audit administrators in each participating jurisdiction.
-          <br />
-          <br />
-          <a
-            href="http://localhost/sample_jurisdiction_filesheet.csv"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            (Click here to view a sample file in the correct format.)
-          </a>
-        </div>
-      </div>
-      <div
-        class="sc-bZQynM ibvdum"
-      >
-        <label
-          class="bp3-file-input"
-        >
-          <input
-            accept=".csv"
-            name="csv"
-            type="file"
-          />
-          <span
-            class="bp3-file-upload-input"
-          >
-            Select a CSV...
-          </span>
-        </label>
-      </div>
-    </div>
-    <div
-      class="bp3-spinner"
-    >
-      <div
-        class="bp3-spinner-animation"
-      >
-        <svg
-          height="50"
-          stroke-width="8.00"
-          viewBox="1.00 1.00 98.00 98.00"
-          width="50"
-        >
-          <path
-            class="bp3-spinner-track"
-            d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
-          />
-          <path
-            class="bp3-spinner-head"
-            d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
-            pathLength="280"
-            stroke-dasharray="280 280"
-            stroke-dashoffset="210"
-          />
-        </svg>
-      </div>
-    </div>
-  </form>
-</div>
-`;
```

Comment:
all of these changes are also suspicious

Reply:
@jonahkagan See above. I'm kind of letting the snapshots fall where they fall right now, and then going back in later with a separate PR to fix them, if that makes sense.